webrender/src/render_backend.rs

```diff
@@ -233,7 +235,10 @@ impl RenderBackend {
                                 self.render()
                             });
 
-                            self.publish_frame(frame, &mut profile_counters);
+                            if self.scene.root_pipeline_id.is_some() {
```

Comment:
I think this if() is unnecessary, right? (since by definition this method is going to result in the root pipeline id being set to a value).